pkg/gecko_sdk/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=gecko_sdk
 PKG_URL=https://github.com/basilfx/RIOT-gecko-sdk
-PKG_VERSION=d381e526d68a2d0c951f37040c1c2e168ac66cd6
+PKG_VERSION=ee01ab57eaaa8805916601dd444d1b63f03037de
```

Comment:
I see there's an update to the gecko sdk. Is this intended here?

Reply:
Yeah, the update is intended, as they provide the utility methods to convert bits to enums.